hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/debug/TestLDBCli.java

```diff
@@ -182,6 +182,30 @@ private static Stream<Arguments> scanTestCases() {
             Named.of("Filter invalid key", Arrays.asList("--filter", "keyName:equals:key9")),
             Named.of("Expect key1-key3", null)
         ),
+        Arguments.of(
+            Named.of(KEY_TABLE, Pair.of(KEY_TABLE, false)),
+            Named.of("Default", Pair.of(0, "")),
+            Named.of("Filter dataSize<2000", Arrays.asList("--filter", "dataSize:lesser:2000")),
+            Named.of("Expect key1-key5", Pair.of("key1", "key6"))
+        ),
+        Arguments.of(
+            Named.of(KEY_TABLE, Pair.of(KEY_TABLE, false)),
+            Named.of("Default", Pair.of(0, "")),
+            Named.of("Filter dataSize<500", Arrays.asList("--filter", "dataSize:lesser:500")),
+            Named.of("Expect empty result", null)
+        ),
+        Arguments.of(
+            Named.of(KEY_TABLE, Pair.of(KEY_TABLE, false)),
+            Named.of("Default", Pair.of(0, "")),
+            Named.of("Filter dataSize>500", Arrays.asList("--filter", "dataSize:greater:500")),
+            Named.of("Expect key1-key5", Pair.of("key1", "key6"))
+        ),
+        Arguments.of(
+            Named.of(KEY_TABLE, Pair.of(KEY_TABLE, false)),
+            Named.of("Default", Pair.of(0, "")),
+            Named.of("Filter dataSize>2000", Arrays.asList("--filter", "dataSize:greater:2000")),
+            Named.of("Expect empty result", null)
+        ),
```

Comment:
should add some -ve test cases somewhere as well

Reply:
I have added some negative test cases in the robot test file now